docker/fix_classpaths.sh

```diff
@@ -7,7 +7,7 @@ sed -i 's/javax.el.jar/jakarta.el.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.enterprise.concurrent-api.jar/jakarta.enterprise.concurrent-api.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.enterprise.concurrent.jar/jakarta.enterprise.concurrent.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.enterprise.deploy-api.jar/jakarta.enterprise.deploy-api.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
-sed -i 's/javax.faces.jar/jakarta.faces.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
+sed -i 's/jakarta.faces.jar/jakarta.faces.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
```

Comment:
If possible can you remove this change. docker/fix_classpaths.sh  is run from Jenkinsfile to correct any javax jar name references in the ts.jte of jakartaeetck bundle before running the platform tck. Mostly we would not require this once we are done with the javax changes.

Reply:
Thanks, fixed now.